sw/host/opentitanlib/src/spiflash/sfdp.rs

```diff
@@ -502,4 +510,18 @@ mod test {
         assert_eq!(sfdp.jedec.density, 128 * 1024 * 1024);
         Ok(())
     }
+
+    // Regression test for https://github.com/lowRISC/opentitan/issues/13477
+    #[test]
+    fn test_bad_header_signature() -> Result<()> {
+        let buf: &[u8] = &[255u8; 256];
```

Comment:
```suggestion
        let buf = &[255u8; 256];
```

Reply:
I actually did this to coerce `buf` into being a `&[u8]` instead of `&[u8; 256]`, so it would match the `impl TryFrom<&[u8]> for Sfdp`.

I don't particularly care either way, I'll just take your suggestion.

Reply:
Oh, I see, makes sense. Sorry for missing that :)